static/js/types/websites.ts

```diff
@@ -40,7 +40,10 @@ export interface Website {
   source: string | null
   starter: WebsiteStarter | null
   metadata: any
-  is_admin?: boolean // eslint-disable-line
+}
+
+export interface WebsiteDetail extends Website {
+  is_admin: boolean // eslint-disable-line
```

Comment:
is this key not present on the `Website` objects always for performance reasons or something? If possible it might be more straightforward to stick with just the one `Website` type unless we really need the separation or need to specifically keep track of when this value is present (rather than just duck-typing when it's truthy).

Reply:
I did a little experiment locally and it doesn't seem to cause any issues to put `is_admin?` back on `Website` and switch `WebsiteDetail` things over to just `Websites`

Reply:
Sure, I'll make the change with `is_admin?`. I do think it is left off the listing for performance though there is probably a way to workaround those issues if we need to